### ISSUE TITLE
Language Dropdown in Header Doesn't Show Focus

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1925]
### Descriptive summary

In Hyrax 2.0.0 rc1, using Nurax, the dropdown to select a language in the header doesn't show focus when tabbing through the interface.

### Rationale

Showing focus when tabbing to actionable elements in the user interface allows the keyboard user to understand that an element can be interacted with and execute actionable items.

https://webaim.org/techniques/keyboard/#problems

### Expected behavior

When tabbing through the header, I expect the language dropwdown to show a visual indicator just like the Hyrax word mark, notification icon, and profile dropdown.

### Actual behavior

The language dropdown does not visually show focus.

### Steps to reproduce the behavior

1. Go to the dashboard
1. Hit tab 
1. Cycle back and forth between Hyrax word mark and profile dropdown

### Related work

https://github.com/samvera/hyrax/issues/497

**Comment by mtribone** [comment]
![focus mov](https://user-images.githubusercontent.com/4163828/31908352-885a0824-b804-11e7-9f80-63b9ee8c48a5.gif)

**Comment by mtribone** [comment]
@adamjarling There might be some discrepancies between Hyrax locally and what is in Nurax. I'm also running Hyrax locally to do my testing and fixing.

**Comment by adamjarling** [comment]
Yea, so local Hyrax should be the source of truth for latest code, correct?   Nurax is updated every week or so?   Or could it be environmental, since "everything" is turned on in Nurax?

**Comment by h-parekh** [comment]
Discussed with @adamjarling, this has been merged with master and can be closed.
